action.yaml

```diff
@@ -119,6 +122,7 @@ runs:
       FLAKEHUB_PUSH_EXTRA_TAGS: ${{ inputs.extra-tags }}
       FLAKEHUB_PUSH_SPDX_EXPRESSION: ${{ inputs.spdx-expression }}
       FLAKEHUB_PUSH_ERROR_ON_CONFLICT: ${{ inputs.error-on-conflict }}
+      FLAKEHUB_PUSH_INCLUDE_OUTPUT_PATHS: ${{ inputs.include-output-paths }}
```

Comment:
Have you tested this in GHA?

I am not 100% sure, but I believe the argument should be given a `default: false` to avoid GHA expanding this to `FLAKEHUB_PUSH_INCLUDE_OUTPUT_PATHS: ` which is... I think, not allowed.

Reply:
I tested this and it seems fine.